sycl/test-e2e/Plugin/sycl-ls-gpu-default-any.cpp

```diff
@@ -1,7 +1,6 @@
 // REQUIRES: gpu
-
-// TODO: Remove unsetting SYCL_DEVICE_FILTER when feature is dropped
-// RUN: env --unset=SYCL_DEVICE_FILTER  --unset=ONEAPI_DEVICE_SELECTOR sycl-ls --verbose >%t.default.out
+// RUN: env --unset=SYCL_DEVICE_FILTER --unset=ONEAPI_DEVICE_SELECTOR sycl-ls --verbose >%t.default.out
+// RUN: %if preview-breaking-changes-supported %{ env --unset=ONEAPI_DEVICE_SELECTOR sycl-ls --verbose >%t.default.out %}
```

Comment:
This new test case does not make sense to me. First of all, we simply overwrite the previous output file before we've even checked it. Second, the command that's run doesn't do anything special. I doubt we've built `sycl-ls` with the `-fpreview-breaking-changes` flag, so running it an extra time if that flag is supported adds nothing new to the test.